doc/appendix-de-kassensichv/reference-tables/reference-tables.md

```diff
@@ -279,3 +279,474 @@ For definitions regarding national laws, please refer to the appropriate appendi
+| `AGENTUR_USTID` | VAT ID| String (15) | Mandatory, `dailyClosingMasterData.Agencies.VATId` |
+
+(`dailyClosingMasterData`is described above in the chapter "Master data module")
+
+##### File: Stamm_USt (vat.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | From `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft  |
+| `UST_SCHLUESSEL` | ID of the VAT rate| Integer | Automatically filled by ft  |
+| `UST_SATZ` | Percentage | Decimal (2) | Automatically filled by ft  |
+| `UST_BESCHR` | Description| String | Automatically filled by ft  |
+
+##### File: Stamm_TSE (tse.csv)
+
+| **Fieldname**            | **Description**          | **Format**          | **ft.input** |
+|----------------------|--------------------------|---------------------|---------------------|
+| `Z_KASSE_ID` | ID of the (closing) cashpoint | String (50) | `ftCashBoxIdentification` |
+| `Z_ERSTELLUNG` | Date of the cashpoint closing | ISO 8601 und RFC3339 date | from `cbReceiptMoment` of the daily closing receipt |
+| `Z_NR` | Nr. of the cashpoint closing | Integer | Automatically filled by ft |
+| `TSE_ID` | TSE ID | Integer | Automatically filled by ft |
+| `TSE_SERIAL` |  Serial number of the TSE (Corresponds according to TR- 03153 section 7.5. to the hash value of the key contained in the certificate; octet string in hexadecimal representation) | String (68) | Automatically filled by ft |
+| `TSE_SIG_ALGO` | The signature algorithm used by the TSE | String (21) | Automatically filled by ft |
+| `TSE_ZEITFORMAT` | The format used by the TSE for the log-time | String (31) | Automatically filled by ft |
+| `TSE_PD_ENCODING` | Text encoding of the process data (UTF-8 or ASCII) | String (5) | Automatically filled by ft |
+| `TSE_PUBLIC_KEY` |  Public key - eventually extracted from the TSE certificate - in base64 encoding | String (512) | Automatically filled by ft |
+| `TSE_ZERTIFIKAT_I` |  First 1,000 characters of the TSE certificate (in base64 encoding) | String (1000) | Automatically filled by ft |
+| `TSE_ZERTIFIKAT_II` | Possibly another 1,000 characters of the certificate (in base64 encoding) | String (1000) | Automatically filled by ft |
+
+#### Cashpoint closing module (DE: Kassenabschlussmodul)
+
+The three structure levels (modules) allow transactions to be separated and grouped, so that they can be automatically posted or transferred to the bookkeeping. The cash register closing therefore has an accounting function, meaning that the data of the respective closing has to be stored digitally within this module. The storage takes place in three files:
```

Comment:
Add: "The mandatory audit proof-storage can be fulfilled with the fiskaltrust add-on-products (AKO - Audicon Kassenarchiv Online) - see: https://fiskaltrust.de/produkte"

Reply:
Thank you, added with this commit:
https://github.com/christain-rogobete/interface-doc/commit/7af9458b52373c2b8eeb41604c695c18cc8f6e78